Source/SupportFunctions/arm_f64_to_q7.c

```diff
@@ -0,0 +1,151 @@
+/* ----------------------------------------------------------------------
+ * Project:      CMSIS DSP Library
+ * Title:        arm_f64_to_q7.c
+ * Description:  Converts the elements of the 64 bit floating-point vector to Q7 vector
+ *
+ * $Date:        18 August 2022
+ * $Revision:    V1.0.0
+ *
+ * Target Processor: Cortex-M and Cortex-A cores
+ * -------------------------------------------------------------------- */
+/*
+ * Copyright (C) 2010-2021 ARM Limited or its affiliates. All rights reserved.
```

Comment:
Should be 2022. 
I have not looked at other files to check if there is a similar issue.